docs/articles/documentation/test-api/actions/action-options.md

```diff
@@ -6,12 +6,48 @@ checked: true
 ---
 # Action Options
 
-This topic describes three types of [test action](index.md) options.
+This topic describes [test action](index.md) options.
 
+* [Basic Action Options](#basic-action-options)
 * [Mouse Action Options](#mouse-action-options)
 * [Click Action Options](#click-action-options)
 * [Typing Action Options](#typing-action-options)
 
+## Basic Action Options
```

Comment:
In fact it works almost for all actions.
`Basic Options` uses for select and pressKey actions.

`Mouse Action Options` extends `Basic Action Options` and is used for some others.

`Client Action Options` extends `Mouse Action Options` etc...

But Maybe this words `Basic action options are used in the t.pressKey, ...` lead to misunderstanding. In fact Basic Options are used in all other actions because `Basic Options` are extended by others options. Maybe we should note it here? What do you think @VasilyStrelyaev ?